pyqtgraph/functions.py

```diff
@@ -662,6 +662,35 @@ def affineSlice(data, shape, origin, vectors, axes, order=1, returnCoords=False,
         return output
 
 
+def interweaveArrays(*args):
```

Comment:
Nice, I was considering suggesting it be in `functions` and generalized like this.

Reply:
There was no reason to have that kind of functionality specific to that ErrorBarItems...figured may as well put it in `functions.py`